### ISSUE TITLE
G rotation different behaviour on android release mode #1662

### ISSUE THREAD

**Issue by vinaysgl** [software-mansion/react-native-svg#1795]
NB Never post screenshots of code. Post copy paste-able code if you include any.
Ignoring this will likely lead to direct closing of the issue.
Please respect the time and energy of open source maintainers.

# Bug

Please provide a clear and concise description of what the bug is.
Include screenshots if needed.
Please test using the latest release of the library, as maybe said bug has been already fixed.

#### Unexpected behavior

If you have unexpected behavior, please create a clean project with the latest versions of react-native and react-native-svg

```bash
react-native init CleanProject
cd CleanProject/
yarn add react-native-svg
cd ios && pod install && cd ..
modify App.js -> run the app
```

If it's still an issue with the latest versions,
please make an sscce in the clean project,
and make it available as a git repo on github.
Or as a https://snack.expo.io/ if expo has the latest version,
and / or the bug exists both in the latest version and the one used in expo and snack.

## Environment info

Run `react-native info` in your terminal and copy the results here. Also, include the *precise* version number of this library that you are using in the project

React native info output:

```bash
 // paste it here
```

Library version: x.x.x

## Steps To Reproduce

Issues without reproduction steps or code are likely to stall.

1. git clone https://github.com/user/rnsvg-bug-repro
2. cd rnsvg-bug-repro
3. yarn
4. yarn ios
5. yarn android
6. ...

Describe what you expected to happen:

1.
2.

## Short, Self Contained, Correct (Compilable), Example

If you are having a problem with some code and seeking help, preparing a Short, Self Contained, Correct Example (SSCCE) is very useful. But what is an SSCCE?

It is all in the name, really. Take a look at each part. The version prepared for others to see should be:

Short (Small) - Minimise bandwidth for the example, do not bore the audience.
Self Contained - Ensure everything is included, ready to go.
Correct - Copy, paste, (compile,) see is the aim.
Example - Displays the problem we are trying to solve.

http://www.sscce.org/

**Comment by vinaysgl** [comment]
https://github.com/react-native-svg/react-native-svg/issues/1662

**Comment by vinaysgl** [comment]
Disabling Hermes seems to solve this issue. 
in android/app/build.gradle

**Comment by bibekpaudyal1** [comment]
I am having a long time issue figuring out the issue behind the problem. i have developed the mobile application where I have use SVG <G > where I have used onPress function but the onpress function in SVG is working fine and well. in android simulator and IOS but when I tried it in android APK it's not clicking out . I have having the hard time figuring out the issue.

1.Here are the ss of the code and SVG i have used in the project
![FD6E05E3-8920-4DEF-86D0-69664D8EEA58](https://user-images.githubusercontent.com/73839358/177243920-7a28e6d6-3c8f-47fb-8a58-49d9d3aa4f0f.jpeg)
![547B1082-283A-4F18-8F20-68215154BB32](https://user-images.githubusercontent.com/73839358/177243928-577f6a0e-236c-40eb-80e9-ec82e75a0f7d.jpeg)
.
